packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -655,3 +655,9 @@ When(
   (folderName: string, count: string) =>
     cy.fileInputUpload(`${folderName}/${randomFileName}`, parseInt(count))
 );
+
+When('I simulate network disconnection', () => {
```

Comment:
Since a network connection failure is not something that a user would intentionally cause (in most scenarios at least) think the verbiage of this util would read more straightforward as:
```suggestion
When('A network failure occurs', () => {
```